.vsts-ci/templates/ci-general.yml

```diff
@@ -14,8 +14,8 @@ steps:
     continueOnError: true
     displayName: Install PowerShell Daily
   - pwsh: |
-      Install-Module -Scope CurrentUser -Force -AllowPrerelease PowerShellGet
-      Install-Module -Scope CurrentUser -Force -AllowPrerelease CompatPowerShellGet
+      Install-Module -Scope CurrentUser -Force -AllowPrerelease -AllowClobber PowerShellGet
```

Comment:
You don't need -AllowClobber for PSGet since it's entirely new cmdlet names

Reply:
It'll scale better once PSGet v3 ships with PowerShell.